homeassistant/const.py

```diff
@@ -385,3 +389,4 @@
 TEMPERATURE = 'temperature'  # type: str
 SPEED_MS = 'speed_ms'  # type: str
 ILLUMINANCE = 'illuminance'  # type: str
+BANDWIDTH = 'bandwidth'  # type: str
```

Comment:
It doesn't make much sense to add constants which are not used. If needed new constants can be added every time.